lib/web_ui/lib/src/engine/text/word_breaker.dart

```diff
@@ -5,11 +5,11 @@
 import '../util.dart';
 import 'word_break_properties.dart';
 
-class _FindBreakDirection {
-  const _FindBreakDirection({required this.step});
+enum _FindBreakDirection {
```

Comment:
Was this automatically detected by a lint? wow!

Reply:
Yes!